### DISCUSSION TITLE
Linguist Language Support

### DISCUSSION THREAD

**Discussion by samchouse** [community/community#11178] [Mobile] [answered]
Add support for `linguist-language` syntax highlighting in `.gitattributes` on mobile.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the feedback @Xenfo — could you share a few more details about how you're using this setting so that I can better understand what kind of outcome you're looking for in the apps?

**Reply by samchouse** [reply]
Let's say I have a file `.eslintrc` and it's written in JSON. GitHub doesn't recognize it as JSON unless I annotate it with linguist in `.gitattributes`. How it looks on mobile is how it looks without `.gitattributes`.

See here:
https://github.com/flux-ui/flux-ui/blob/main/.gitattributes
https://github.com/flux-ui/flux-ui/blob/main/.eslintrc

**Reply by brianlovin** [reply]
Today I learned! Thanks for the links and context @Xenfo, I'll open a tracking issue for our team to take a look here, although I'm not exactly sure when we'll be able to get this on the roadmap 😊
